grpc/CMakeLists.txt

```diff
@@ -15,15 +15,22 @@ if(NOT RSYNC)
   message(SEND_ERROR "Cannot find rsync.")
 endif(NOT RSYNC)
 
+set(GRPC_BUILD_SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/grpc_build")
```

Comment:
You can also substitute the `${CMAKE_CURRENT_BINARY_DIR}/grpc_build` with the `${GRPC_BUILD_SOURCE_DIR}` in L91, 97 and 109.

Reply:
Thanks. Updated 👍